cmd_events.go

```diff
@@ -82,6 +82,10 @@ func doEvents() error {
 			Icon(aw.IconWorkflow)
 	}
 
+	// Get our current timezone
+	zone, _ := time.Now().Zone()
+	loc, _ := time.LoadLocation(zone)
```

Comment:
I'm not a fan of ignoring errors.

Is this code necessary? Can't `time.Local` be used instead?

Reply:
Not sure how I missed that on the doc page